NEWS.rst

```diff
@@ -40,6 +40,7 @@ possible, especially the following contributors:
 - Rob Miller
 - Sergio Valqui
 - Sujan Dulal (first contribution)
+- Hielke Walinga (first contribution)
```

Comment:
This is usually alphabetical by first name (same as the CONTRIB file)

Reply:
OK